ts/features/itwallet/credentials/store/selectors/index.ts

```diff
@@ -37,3 +39,12 @@ export const itwCredentialsTypesSelector = createSelector(
   itwCredentialsByTypeSelector,
   credentials => Object.keys(credentials)
 );
+
+export const selectFiscalCodeFromEid = createSelector(
+  itwCredentialsSelector,
+  credentials =>
+    pipe(
+      credentials.eid,
+      O.fold(() => "", getFiscalCodeFromCredential)
+    )
```

Comment:
```suggestion
    pipe(
      credentials.eid,
      O.map(getFiscalCodeFromCredential),
      O.getOrElse(() => "")
    )
```

I think that adding a map makes this function easier to read. What do you think?

Reply:
Addressed in 687a3d2fdf1ed15a458369b5de20e0994a1595d0.